control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1920,6 +1931,54 @@ func (r *HostedControlPlaneReconciler) reconcileOIDCRouteResources(ctx context.C
 	return err
 }
 
+func (r *HostedControlPlaneReconciler) reconcileImageContentSourcePolicy(ctx context.Context, hcp *hyperv1.HostedControlPlane) error {
+	imageContentSourceCRD := manifests.ImageContentSourcePolicy()
```

Comment:
nit: technically not a crd, but a cr... `imageContentSourcePolicy` seems a better name.

Reply:
changed!